src/components/SelectionList/UserListItem.tsx

```diff
@@ -18,45 +29,76 @@ function UserListItem({item, textStyles, alternateTextStyles, showTooltip, style
 
     return (
-        <>
-            {!!item.icons && (
+        <BaseListItem
+            item={item}
+            wrapperStyle={[styles.flex1, styles.justifyContentBetween, styles.sidebarLinkInner, styles.userSelectNone, styles.peopleRow, isFocused && styles.sidebarLinkActive]}
+            selectMultipleStyle={[
+                StyleUtils.getCheckboxContainerStyle(20),
+                styles.mr3,
+                item.isSelected && styles.checkedContainer,
+                item.isSelected && styles.borderColorFocus,
+                item.isDisabled && styles.cursorDisabled,
+                item.isDisabled && styles.buttonOpacityDisabled,
+            ]}
+            isFocused={isFocused}
+            isDisabled={isDisabled}
+            showTooltip={showTooltip}
+            canSelectMultiple={canSelectMultiple}
+            onSelectRow={onSelectRow}
+            onDismissError={onDismissError}
+            shouldPreventDefaultFocusOnSelectRow={shouldPreventDefaultFocusOnSelectRow}
+            rightHandSideComponent={rightHandSideComponent}
+            keyForList={item.keyForList}
+        >
+            {(hovered) => (
                 <>
-                    {item.shouldShowSubscript ? (
-                        <SubscriptAvatar
-                            mainAvatar={item.icons[0]}
-                            secondaryAvatar={item.icons[1]}
-                            showTooltip={showTooltip}
-                            backgroundColor={isHovered && !isFocused ? hoveredBackgroundColor : subscriptAvatarBorderColor}
-                        />
-                    ) : (
-                        <MultipleAvatars
-                            icons={item.icons ?? []}
+                    {!!item.icons && (
+                        <>
+                            {item.shouldShowSubscript ? (
+                                <SubscriptAvatar
+                                    mainAvatar={item.icons[0]}
+                                    secondaryAvatar={item.icons[1]}
+                                    showTooltip={showTooltip}
+                                    backgroundColor={hovered && !isFocused ? hoveredBackgroundColor : subscriptAvatarBorderColor}
+                                />
+                            ) : (
+                                <MultipleAvatars
+                                    icons={item.icons ?? []}
+                                    shouldShowTooltip={showTooltip}
+                                    secondAvatarStyle={[
+                                        StyleUtils.getBackgroundAndBorderStyle(theme.sidebar),
+                                        isFocused ? StyleUtils.getBackgroundAndBorderStyle(focusedBackgroundColor) : undefined,
+                                        hovered && !isFocused ? StyleUtils.getBackgroundAndBorderStyle(hoveredBackgroundColor) : undefined,
```

Comment:
I see that it was that way earlier, but could it be simplified somehow?

Reply:
I guess we could only simplify it like below (got rid of ternary operator):

```javascript
isFocused && StyleUtils.getBackgroundAndBorderStyle(focusedBackgroundColor),
hovered && !isFocused && StyleUtils.getBackgroundAndBorderStyle(hoveredBackgroundColor),
```

what do you think?

Reply:
So then I think it can stay as it is. Leaving it up to you

Reply:
Ok. Let's leave it. Thanks - marking as resolved.